admin_guide/idling_applications.adoc

```diff
@@ -16,6 +16,12 @@ toc::[]
 As an {product-title} administrator, you can idle applications in order to
 reduce the consumption of resources. 
 
+[NOTE]
+====
+This works when using the default HAProxy router. You will need to configure
```

Comment:
Erm... you can't configure the F5 to do this.  It requires a code change, and we need to work out how to make it do it (or make it use the service IP... but it doesn't have access to the service network, so it's ugly).

How about "Automatic unidling by a router is currently only supported by the default HAProxy router"

But it really does belong at the bottom since that's the bit that mentions routes.

Should we add a caveat to the F5 section too?  So that people are informed when they pick the F5?